src/make-comment.ts

```diff
@@ -33,25 +33,23 @@ const makeComment = async (message: string, commentData: CommentData): Promise<v
     });
 
     const botComment = comments.data.find((comment) => comment.body?.includes('<!-- coverage-action-comment -->'));
-    const customMessage = getInput('customMessage');
 
     console.log(botComment);
```

Comment:
Please remove `console.log `

Reply:
We actually need the `console.log` because it ends up logging this information to the CI so that the people can see the comment there as well.